modules/api/src/test/resources/ocl_test.yaml

```diff
@@ -0,0 +1,116 @@
+  PbKJHq7DRTRXT60avtuTRdbrFJI3mSZhNOqYjVbd99YyK1QKWzEqSWrE0k07U60uPaelflMzaaeu1KBuurHSsn572I1KWy2joX5ZBfWbS/VEt50H5P6aL4JxTuyJ/+QCNPX4PWF3Q8Xe1eF9FsLdD2VaOnaP2hWvs+zI58/7i3vH3nRFtDZpyTUNaZkON5XnBNsp
+  8lrmDMrpvBr+b6pUl+4XbkQdndqnzYGzfuJm1JmIWimIbe6dndd/bk7gVce/cJdo3uIeLJl7+I2xTnPek67mjtDeppE7b03Ov+kSfDe3JweW53njxeGfXkaz28VeYd86+af/H8a7hgJKaebILaFzakLfxyfQLTxVB6K1K9KQAAAABJRU5ErkJggg==
+# Reserved for CSP, aws,azure,ali,huawei and ...
+cloudServiceProvider:
+  name: huawei
+  regions:
+    - name: cn-southwest-2
+      area: Asia Pacific
+    - name: cn-north-4
+      area: North America
+billing:
+  # The business model(`flat`, `exponential`, ...)
+  model: flat
+  # The rental period (`daily`, `weekly`, `monthly`, `yearly`)
+  period: monthly
+  # The billing currency (`euro`, `usd`, ...)
+  currency: euro
+# The flavor of the service, the @name/@version/@flavor can locate the specific service to be deployed.
+flavors:
+  - name: 3-node-without-zookeeper
+    # The fixed price during the period (the price applied one shot whatever is the service use)
+    fixedPrice: 20
+    # Properties for the service, which can be used by the deployment.
+    property:
+      node: 3
+      zookeeper: false
+  - name: 5-node-with-zookeeper
+    # The fixed price during the period (the price applied one shot whatever is the service use)
+    fixedPrice: 30
+    # Properties for the service, which can be used by the deployment.
+    property:
+      node: 5
+      zookeeper: true
+deployment:
+  # kind, Supported values are terraform, pulumi, crossplane.
+  kind: terraform
+  # Context for deployment: the context including some kind of parameters for the deployment, such as fix,variable.
+  # - env: The value of the fix parameters are defined by the ISV with the @value at the initial time.
+  # - variable: The value of the variable parameters are defined by the user on the console.
+  # The parameters will be used to generate the API of the managed service.
+  context:
+    - name: HW_REGION_NAME
+      description: Your access key.
+      kind: env
+      type: string
+      mandatory: true
+      validator: null
+    - name: HW_ACCESS_KEY
+      description: Your security key.
+      kind: env
+      type: string
+      mandatory: true
+      validator: null
+    - name: HW_SECRET_KEY
+      description: The vpc you want to deploy, If null, A new vpc will be created.
+      kind: env
+      type: string
+      mandatory: false
+      validator: length(1-256)
+    - name: secgroup_id
+      description: The secgroup id.
+      kind: variable
+      type: string
+      mandatory: false
+      validator: length(1-256)
+  deployer: |
+    variable "secgroup_id" {}
+    data "huaweicloud_availability_zones" "myaz" {}
+    data "huaweicloud_compute_flavors" "myflavor" {
+        availability_zone = data.huaweicloud_availability_zones.myaz.names[0]
+        performance_type  = "normal"
+        cpu_core_count    = 2
+        memory_size       = 4
```

Comment:
The context of OCL2.0 can refer to the context in README.md, the TF script can use some free resource, such as vpc , vpc_subnet, security_greoups.

Reply:
ok,i think so